suites/selftest/ts/package.xml

```diff
@@ -5,6 +5,30 @@
     <description>TE Self Tests</description>
     <author mailto="te-maint@oktetlabs.ru"/>
 
+    <script id="default_arg_test" name="default_args">
```

Comment:
I strongly dislike that it is named script since it clashes with existing script and extension of the element will extend it in another place. Also it looks really misleading.

 I'd name it a 'script-template' or something like this. It should inherit the same structure as a script and add 'id'

Reply:
Changed to 'script-template'.